configobject/history/history.go

```diff
@@ -34,23 +34,19 @@ var mysqlObservers = struct {
 }
 
 var historyCounter = struct {
-	state            int
-	notification     int
-	usernotification int
-	downtime         int
-	comment          int
-	flapping         int
-	acknowledgement  int
+	state            int64
+	notification     int64
+	usernotification int64
+	downtime         int64
+	comment          int64
+	flapping         int64
+	acknowledgement  int64
```

Comment:
What datatype did we agree on for counters?

Reply:
uint64